### DISCUSSION TITLE
Is it possible to read the version number from a file instead of Git tag?

### DISCUSSION THREAD

**Discussion by sithembiso** [goreleaser/goreleaser#3373] [Q&A] [answered]
Hi, 
I have an existing project I am trying to migrate to use Goreleaser, but the version number is not tracked on Git Tags. Instead, it's written into a file. Is it to use Git tags to track versions?

**Comment by caarlos0** [comment, marked as answer]
no, but you can provide it as an environment variable: https://goreleaser.com/cookbooks/set-a-custom-git-tag/?h=goreleaser_
